client/src/SessionsList/SessionsListContainer.js

```diff
@@ -29,7 +34,36 @@ export function SessionsListContainer({ listIdentifier, setListIdentifier }) {
 
   return (
     <>
-      <SessionsList listId={listIdentifier} sessions={sessions || []} />
+      <SessionsList
+        listId={listIdentifier}
+        onDeleteSession={(session) => {
+          setSesssionToDelete(session)
+          setConfirmationDialogOpen(true)
+        }}
+        sessions={sessions || []}
+      />
+      {sesssionToDelete && (
+        <Confirmation
+          cancel={() => {
+            setSesssionToDelete(null)
+            setConfirmationDialogOpen(false)
+          }}
+          confirm={() => {
+            deleteSession(sesssionToDelete.id)
+            setConfirmationDialogOpen(false)
+            setSesssionToDelete(null)
+            // TODO replace this hacka-hacka reload with something better
+            window.location.reload()
```

Comment:
you should at least refetch or invalidate session list query (in `useSessionsList`)

that should be the lowest effort solution instead of full reload

Reply:
done